sanitiser/_geo.js

```diff
@@ -46,26 +47,25 @@ function sanitize_bbox( clean, param ) {
     return;
   }
 
-  var bbox = [];
   var bboxArr = param.split( ',' );
 
   if( Array.isArray( bboxArr ) && bboxArr.length === 4 ) {
 
-    bbox = bboxArr.filter( function( latlon, index ) {
-      latlon = parseFloat( latlon, 10 );
-      return !(lat_lon_checks[(index % 2 === 0 ? 'lon' : 'lat')].is_invalid( latlon ));
+    var bbox = bboxArr.map( function ( latlon ){
+      return parseFloat( latlon, 10 );
```

Comment:
`parseFloat()` doesn't accept a second arg like the confusingly mandatory `radix` one from `parseInt()`